### PR TITLE
Adding `audio-worklet/basic` with basic demos

### PR DESCRIPTION
**Pull request by hoch**
PTAL.

### CODE REVIEW

File: audio-worklet/basic/audio-param-modulation.html [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
+    <script src="../lib/audio-worklet-helper.js"></script>
+  </head>
+  <body>
+    <a href="../index.html" class="link-small">&#171; AudioWorklet home</a>
+    <h1>
+      Demo: AudioParam Modulation
+    </h1>
+    <script>
+    // TODO(hoch): remove this GC hack when it is implemented properly.
+    let paramAmp;
```
Comment (rtoy) [inline audio-worklet/basic/audio-param-modulation.html]: Does this mean that GC removes paramAmp even though it's in use?

File: audio-worklet/basic/js/bit-crusher.js [hunk 2]
```diff
@@ -1,40 +1,46 @@
...
-      maxValue: 16 
-    }, {
-      name: 'frequencyReduction',
-      defaultValue: 0.5,
-      minValue: 0,
-      maxValue: 1
-    }];
+  static get parameterDescriptors() {
+    return [
+      {name: 'bitDepth', defaultValue: 12, minValue: 1, maxValue: 16}, {
```
Comment (rtoy) [inline audio-worklet/basic/js/bit-crusher.js]: Is this clang-format?  If so, I really, Really, REALLY hate how this is formatted.

File: audio-worklet/basic/audio-param-modulation.html (hunk 1 above)
Comment (hoch) [inline audio-worklet/basic/audio-param-modulation.html]: That's what the current prototype does.

File: audio-worklet/basic/js/bit-crusher.js (hunk 2 above)
Comment (hoch) [inline audio-worklet/basic/js/bit-crusher.js]: Yeap. Well, we decided to bite the bullet. Didn't we? :)

**Comment by hoch** [comment]
Actually I made changes in `.gitignore` to exclude `.DS_Store`. So this PR is removing them from the repository.

File: audio-worklet/basic/js/bit-crusher.js (hunk 2 above)
Comment (rtoy) [inline audio-worklet/basic/js/bit-crusher.js]: Yes, we did. I just wish clang-format had some consistency.  If some item is spread out over several lines, do it for all.  Or not.  Just not some random mix.

But that's life, so this is fine.

**Pull request merged**